rclcpp/include/rclcpp/executor.hpp

```diff
@@ -102,31 +109,26 @@ class Executor
   {
     this->add_node(node);
     // non-blocking = true
-    std::shared_ptr<AnyExecutable> any_exec;
+    AnyExecutable::SharedPtr any_exec;
```

Comment:
The variable can be declared in the `while` statement to narrow its scope.

Using `auto` would be an option as well.